### PR TITLE
shuffle true warning

### PR DESCRIPTION
**Pull request by augustebaum**
- **refactor(train_test_split): Remove dead tests**
- **feat(train_test_split): Add "shuffle is True" warning**

### CODE REVIEW

**Review by rouk1 — APPROVED** [review]
👍
